tests/variantstudy/model/test_variant_model.py

```diff
@@ -190,3 +198,104 @@ def test_commands_service(
+        user2 = User(id=3, name="jane.doe", type="users")
+        db.session.add(user2)
+        db.session.commit()
+
+        user2_params = RequestParameters(
+            user=JWTUser(
+                id=user2.id,
+                impersonator=user2.id,
+                type="users",
+                groups=[JWTGroup(id="writers", name="writers", role=RoleType.WRITER)],
+            )
+        )
+
+        # Generate a variant on a study that allow other user to edit it
+        variant_study = variant_study_service.create_variant_study(root_study_id, "new variant", params=owner_params)
+        study_version = StudyVersion.parse(variant_study.version)
+        variant_id = variant_study.id
+
+        # Create two new commands on the existing variant
+        command_6 = CommandDTO(action="update_comments", args={"comments": "new comment"}, study_version=study_version)
+        command_7 = CommandDTO(
+            action="update_comments", args={"comments": "another new comment"}, study_version=study_version
+        )
+
+        variant_study_service.append_command(variant_id, command_6, params=owner_params)
+        variant_study_service.append_command(variant_id, command_7, params=user2_params)
+
+        # Make sure there are commands generated by both users
+        commands = variant_study_service.get_commands(variant_id, params=owner_params)
+        assert len(commands) == 2
+
+        # Make sure their `user_name` and `updated_at` attributes are not None
+        for command in commands:
+            assert command.user_name and command.updated_at
+
+        # Make sure commands has not the same author
+        assert commands[0] != commands[1]
+        assert commands[0].user_name == "john.doe"
+        assert commands[1].user_name == "jane.doe"
+
+    @pytest.mark.parametrize("root_study_id", [False], indirect=True)
+    @with_db_context
+    def test_command_same_author(
+        self,
+        jwt_user: JWTUser,
+        variant_study_service: VariantStudyService,
+        root_study_id: str,
+    ):
+        """
+        Test if multiple commands of the same user have the same author
+        Test if database was called for each command
+        """
+        owner_params = RequestParameters(user=jwt_user)
+
+        # Generate a variant on a study that allow other user to edit it
+        variant_study = variant_study_service.create_variant_study(root_study_id, "new_variant", params=owner_params)
+
+        commands = []
+
+        # Create two new commands on the existing variant
+        for index in range(5):
+            commands.append(
+                CommandDTO(
+                    action="update_comments",
+                    args={"comments": f"new comment {index}"},
+                    study_version=StudyVersion.parse(variant_study.version),
+                )
+            )
+
+        variant_study_service.append_commands(variant_study.id, commands, params=owner_params)
+
+        with patch.object(
```

Comment:
It works well but it doesn't really check that we call the DB once, it checks we call the method. If you can change this to really change the DB it would be nice. Otherwise it's not really a big deal